internal/image/attestation.go

```diff
@@ -81,6 +82,10 @@ func (a *attestation) getBuildSCM() string {
 	uri := "" //https://github.com/joejstuart/ec-cli.git"
 	if len(a.Predicate.Materials) == 1 {
 		uri = a.Predicate.Materials[0].Uri
+		// If our URI has 'git+https' as the scheme, we will strip the "git+" portion.
+		if strings.HasPrefix(uri, "git+") {
+			uri = strings.TrimLeft(uri, "git+")
+		}
```

Comment:
nit: this could be:
```suggestion
		// If our URI has 'git+https' as the scheme, we will strip the "git+" portion.
		uri = strings.TrimPrefix(a.Predicate.Materials[0].Uri, "git+")
```
(if we want one conditional less)